common.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+ERRORED=false
+image_location=${RIPSAW_CI_IMAGE_LOCATION:-quay.io}
+image_account=${RIPSAW_CI_IMAGE_ACCOUNT:-rht_perf_ci}
+es_server=${ES_SERVER:-foo.esserver.com}
+es_port=${ES_PORT:-80}
+echo "using container image location $image_location and account $image_account"
+
+function wait_clean {
+  if [[ `kubectl get benchmarks.ripsaw.cloudbulldozer.io --all-namespaces` ]]
+  then
+    kubectl delete benchmarks -n my-ripsaw --all --ignore-not-found
+  fi
```

Comment:
Not doing this could actually cause a deadlock within Kubernetes because of how Kubernetes deletes child resources in a namespace.

Since the benchmarks are a CR, only the benchmark-operator knows how to delete them. However, I don't believe our CRs are strictly owned by the operator (i.e. no ownerReference). When the namespace is deleted, Kubernetes will try to delete the benchmarks and the operator deployment at the same time. Most of the time this causes the operator pod to get killed before all the benchmark CRs have been deleted and so the namespace could get stuck in a terminating state